CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ Change log
 life-sync uses [PVP Versioning][1].
 The change log is available [on GitHub][2].
 
+# 1.0.4
```

Comment:
The last released version on Hackage is `1.0.1` so everything above this version should go to `1.0.2` (or 1.1.0 when the first breaking change is introduced).